framework/uicomponents/qml/Muse/UiComponents/tests/menuview_tests.cpp

```
@@ -0,0 +1,111 @@
+};
+}
+
+TEST(MenuViewTests, OverflowingTopLevelMenuOpensLeftOfCursor)
+{
+    QScreen* screen = QGuiApplication::primaryScreen();
+    ASSERT_NE(screen, nullptr);
+
+    const QRect availableGeometry = screen->availableGeometry();
+    ASSERT_GT(availableGeometry.width(), 400);
+
+    QQuickWindow parentWindow;
```

Clang (14.0.6)

[warning] variable 'parentWindow' is not initialized

(cppcoreguidelines-init-variables)